### PR TITLE
Feature/storeis

### PR DESCRIPTION
**Pull request by MALWICK**

### CODE REVIEW

File: src/Components/Atoms.tsx
Comment (kadjialan) [inline src/Components/Atoms.tsx:1]: could create  a folder for atoms inside  which you could create a file for react icons for better separation of concerne

File: src/app/modal/FeedCard.tsx
Comment (kadjialan) [inline src/app/modal/FeedCard.tsx:1]: why join  the styling with the code when you can create them in your components an import them

File: src/app/modal/Postbox.tsx
Comment (kadjialan) [inline src/app/modal/Postbox.tsx:1]: styles join together with code

File: src/app/modal/Stories.tsx
Comment (kadjialan) [inline src/app/modal/Stories.tsx:1]: what is the need for this file?

**Review by kadjialan — CHANGES_REQUESTED** [review]

File: src/Components/Atoms.tsx
Comment (MALWICK) [reply src/Components/Atoms.tsx:1]: i did create a folder for atoms and a file for iconatoms.jsx

File: src/app/modal/FeedCard.tsx
Comment (MALWICK) [reply src/app/modal/FeedCard.tsx:1]: i have exported the styling they are no more in our files

File: src/app/modal/Postbox.tsx
Comment (MALWICK) [reply src/app/modal/Postbox.tsx:1]: i have separated the styling

File: src/app/modal/Stories.tsx
Comment (MALWICK) [reply src/app/modal/Stories.tsx:1]: i deleted those unwanted icons

**Review by kadjialan — DISMISSED** [review]

File: src/app/modal/Stories.tsx
Comment (kadjialan) [inline src/app/modal/Stories.tsx:1]: why submit a code with comments?

File: src/assets/1.jpeg
Comment (kadjialan) [inline src/assets/1.jpeg:1]: i think a folder image will be better to contain the images

**Review by kadjialan — CHANGES_REQUESTED** [review]

**Review by kadjialan — APPROVED** [review]

**Pull request closed without merging**